service/cart_http_test.go

```diff
@@ -0,0 +1,218 @@
+package service
+
+import(
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"joshsoftware/go-e-commerce/db"
+	"net/http"
+	"errors"
+)
+
+type Cart struct {
+	cartID int
+	productID int
+	quantity int
+}
+
+var testCart = Cart {
```

Comment:
where testCart is used?

Reply:
Earlier i used different approach but then forgot to remove after using current approach. Now removed